tests/components/recorder/test_util.py

```diff
@@ -1117,3 +1128,106 @@ async def test_resolve_period(hass: HomeAssistant) -> None:
             }
         }
     ) == (now - timedelta(hours=1, minutes=25), now - timedelta(minutes=25))
+
+
+NonRetryable = OperationalError(None, None, None, "")
```

Comment:
According to the static typechecker, the third constructor parameter (`orig`) must be a `BaseException` and cannot be `None`.

Also, the fourth parameter (`hide_parameters`) must be a Boolean.